content/400-reference/200-api-reference/100-prisma-schema-reference.mdx

```diff
@@ -1169,7 +1169,7 @@ Defines a single-field ID on the model.
 > @id
 > ```
 
-> **Note**: In version 3.5.0 and later, with the `"extendedIndexes"` Preview feature enabled, the signature is:
+> **Note**: In versions `4.0.0` and later, and in versions `3.5.0` and later with the `extendedIndexes` Preview feature enabled, the signature is:
```

Comment:
I think this might actually justify switching the order around:
First the new signature, then the one from before 4.0.0 (or between 3.5.0 and 4.0.0 without `extendedIndexes`).
We should list the new default first, and then list the special cases from the past.

(Same for the other changes in this file)

Reply:
Agree, I've moved the signatures around